Cargo install fails with latest on Ubuntu 22.04

Cargo build fails on default Ubuntu 22.04

The Cargo / rustc version is 1.61.0.

Fails when running:

```
cargo install just
```

The failure is:

```
error[E0015]: cannot call non-const fn `std::sync::Mutex::<interrupt_handler::InterruptHandler>::new` in statics
  --> /home/fquinn/.cargo/registry/src/github.com-1ecc6299db9ec823/just-1.10.0/src/interrupt_handler.rs:17:48
   |
17 |     static INSTANCE: Mutex<InterruptHandler> = Mutex::new(InterruptHandler::new());
   |                                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: calls in statics are limited to constant functions, tuple structs and tuple variants

For more information about this error, try `rustc --explain E0015`.
```

And it happens for both 1.10.0 and 1.11.0 (1.9.0 works fine) if specified explicitly on cargo command line so thinking it may be related to #1442

--- a/.github/workflows/ci.yaml
+++ b/.github/workflows/ci.yaml
@@ -51,14 +51,6 @@ jobs:
         sudo apt-get install ripgrep
         ./bin/forbid
 
-    - name: cargo-msrv
-      uses: baptiste0928/cargo-install@v1
-      with:
-        crate: cargo-msrv
-
-    - name: Verify MSRV
-      run: cargo-msrv verify
-
   pages:
     runs-on: ubuntu-latest
     permissions:

--- a/.github/workflows/ci.yaml
+++ b/.github/workflows/ci.yaml
@@ -50,6 +50,15 @@ jobs:
         sudo apt-get update
         sudo apt-get install ripgrep
         ./bin/forbid
+
+    - name: cargo-msrv
+      uses: baptiste0928/cargo-install@v1
+      with:
+        crate: cargo-msrv
+
+    - name: Verify MSRV
+      run: cargo-msrv verify
+
   pages:
     runs-on: ubuntu-latest
     permissions:

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -12,6 +12,7 @@ autotests = false
 categories = ["command-line-utilities", "development-tools"]
 keywords = ["command-line", "task", "runner", "development", "utility"]
 exclude = ["/book", "/icon.png", "/screenshot.png", "/www"]
+rust-version = "1.63"
 
 [workspace]
 members = [".", "bin/ref-type", "bin/generate-book", "bin/update-contributors"]

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,17 +1,17 @@
 [package]
 name = "just"
 version = "1.11.0"
-description = "🤖 Just a command runner"
 authors = ["Casey Rodarmor <casey@rodarmor.com>"]
-license = "CC0-1.0"
-homepage = "https://github.com/casey/just"
-repository = "https://github.com/casey/just"
-readme = "crates-io-readme.md"
-edition = "2021"
 autotests = false
 categories = ["command-line-utilities", "development-tools"]
-keywords = ["command-line", "task", "runner", "development", "utility"]
+description = "🤖 Just a command runner"
+edition = "2021"
 exclude = ["/book", "/icon.png", "/screenshot.png", "/www"]
+homepage = "https://github.com/casey/just"
+keywords = ["command-line", "task", "runner", "development", "utility"]
+license = "CC0-1.0"
+readme = "crates-io-readme.md"
+repository = "https://github.com/casey/just"
 rust-version = "1.63"
 
 [workspace]

